src/vos/vos_pool.c

```diff
@@ -909,6 +915,18 @@ vos_pool_ctl(daos_handle_t poh, enum vos_pool_opc opc)
 		if (pool->vp_vea_info != NULL)
 			vea_flush(pool->vp_vea_info, false);
 		break;
+	case VOS_PO_CTL_SET_POLICY:
+		if (param == NULL)
+			return -DER_INVAL;
+
+		p = (struct policy_desc_t *)param;
```

Comment:
No cast should be required here.
```suggestion
		p = param;
```